wavesexchange_topic/src/lib.rs

```diff
@@ -409,6 +479,27 @@ mod parse_and_format {
             })
         }
 
+        fn query_get_vec<'a>(url: &'a Url, key: &str) -> Option<Vec<Cow<'a, str>>> {
+            let mut vals: Vec<Cow<_>> = vec![];
+
+            for i in url.query_pairs().into_iter() {
```

Comment:
Instead of `for i in pairs` + `i.0` and `i.1` better to write `for (a, b) in pairs`.